packages/client/src/runtime/utils/datmodelBuilder.ts

```diff
@@ -37,12 +37,10 @@ export function model(name: string, fields: DMMF.Field[]): DMMF.Model {
   }
 }
 
-export function baseDmmf({ models }: { models: DMMF.Model[] }) {
-  return new DMMFDatamodelHelper({
-    datamodel: {
-      models,
-      enums: [],
-      types: [],
-    },
+export function runtimeDataModel({ models }: { models: DMMF.Model[] }): RuntimeDataModel {
```

Comment:
Is this just for tests? If so could it be moved out of the runtime folder utils?

Reply:
moved to `src/testUtils`